cli/file_fetcher.rs

```diff
@@ -308,21 +294,15 @@ impl FileFetcher {
     let (source, content_type) = get_source_from_data_url(specifier)?;
     let (media_type, _) = map_content_type(specifier, Some(&content_type));
 
-    let local =
-      self
-        .http_cache
-        .get_cache_filename(specifier)
-        .ok_or_else(|| {
-          generic_error("Cannot convert specifier to cached filename.")
-        })?;
     let mut headers = HashMap::new();
     headers.insert("content-type".to_string(), content_type);
+    // TODO(nayeemrmn): Don't use the fs cache for data URLs:
+    // https://github.com/denoland/deno/issues/18260#issuecomment-1494791622.
```

Comment:
Is there a blocker to addressing this right now?

Reply:
https://github.com/lucacasonato/esbuild_deno_loader/pull/54 hasn't been incorporated in fresh yet I don't think.

Reply:
@nayeemrmn thanks for linking that issue, it should now be used by Fresh, so can we address this TODO before landing?